get-to-know-spdockit/best-practices-screen.md

```diff
@@ -18,6 +18,8 @@ The following options are available:
 * Personalize the Best Practices dashboard by hiding reports you do not wish to track or adjust some of the built-in values using the [Customize Best Practices](#internal/how-to/best-practices/personalize-best-practices-reports) button.
 * [Creating new alerts](#internal/how-to/subscriptions-and-alerts/create-new-alert) for the configuration settings you wish to track. If you choose to create an alert for a specific best practice report, you will receive this report each time SPDocKit detects either an error or a warning.
 
+In version 7.2 we expanded our filters to be aplicable on the Best Practices screen as well. It means that now, you can more easily manage your snapshots. But most importantly we improved the way they are managed across the application - so the experience using the application will be much smoother.
```

Comment:
I think this can be said in a way that will be more meaningful to most users. I really don't think they'll care much about this from the perspective of the filters
Maybe something along the lines of:
*In version 7.2 we streamlined the selection of farms and snapshots by including those filters on the Farm Explorer and Best Practices tabs and persisting the selection across all tabs. This should remove the need to switch back and forth between the snapshots list and the other tabs in order to change the viewed snapshot data on those two tabs . We hope this results in a much more enjoyable experience for you.*

Something like this tells the story of how this benefits the user better I think.